extensions/pdh/PerformanceDataMonitor.cpp

```diff
@@ -45,9 +45,15 @@ core::Property PerformanceDataMonitor::CustomPDHCounters(
 core::Property PerformanceDataMonitor::OutputFormatProperty(
     core::PropertyBuilder::createProperty("Output Format")->
     withDescription("Format of the created flowfiles")->
-    withAllowableValue<std::string>(JSON_FORMAT_STR)->
-    withAllowableValue(OPEN_TELEMETRY_FORMAT_STR)->
-    withDefaultValue(JSON_FORMAT_STR)->build());
+    withAllowableValue(PRETTY_JSON_FORMAT_STR)->
+    withAllowableValue(COMPACT_JSON_FORMAT_STR)->
+    withAllowableValue(PRETTY_OPEN_TELEMETRY_FORMAT_STR)->
+    withAllowableValue(COMPACT_OPEN_TELEMETRY_FORMAT_STR)->
```

Comment:
it seems `withAllowableValues` is more prevalent than `withAllowableValue`

Reply:
good idea, changed it in [26802f7](https://github.com/martinzink/nifi-minifi-cpp/commit/26802f77ef539757fcedbffea0f596b2ef0cc828)